### ISSUE TITLE
Upload a single MultiWOZ dialogue file and get many transcripts

### ISSUE THREAD

**Issue by aidanmrli** [Dhairya-Khara/TLI-Voiceflow-Project#83]
A dialogue.json file contains hundreds of transcripts. Idea: by uploading a single dialogue file and indicating it is a MultiWOZ dialogue file, we get all these transcripts at once, and are able to visualize each of them on their own as well as collectively (as if we uploaded individual transcripts one by one).

**Comment by aidanmrli** [comment]
Aidan Thurs 1 Dec: Managed to enable proper updating of user's intents such that we can upload one dialogue with multiple transcripts in one go, and see the aggregate visualization of all the transcripts with the Visualize All function. Ensured this adheres to the Open-Closed Principle because we still support single transcripts like we did previously, but we also have the ability to upload multiple transcripts in one file. This also agrees with the Liskov Substitution principle because we can substitute a single transcript into a multiple transcript format.
